composer/algorithms/blurpool/blurpool.py

```diff
@@ -18,36 +18,47 @@
 
 
 def apply_blurpool(model: torch.nn.Module,
-                   optimizers: Optional[Optimizers] = None,
                    replace_convs: bool = True,
                    replace_maxpools: bool = True,
-                   blur_first: bool = True) -> None:
+                   blur_first: bool = True,
+                   optimizers: Optional[Optimizers] = None) -> torch.nn.Module:
     """Add anti-aliasing filters to the strided :class:`torch.nn.Conv2d` and/or :class:`torch.nn.MaxPool2d` modules
     within `model`.
 
-    Must be run before the model has been moved to accelerators and before
-    the model's parameters have been passed to an optimizer.
+    These filters increase invariance to small spatial shifts in the input
+    (`Zhang 2019 <http://proceedings.mlr.press/v97/zhang19a.html>`_).
 
     Args:
-        model: model to modify
-        optimizers (Optimizers, optional): Existing optimizers bound to ``model.parameters()``.
-            All optimizers that have already been constructed with,
-            ``model.parameters()`` must be specified here so they will optimize
-            the correct parameters.
-
-            If the optimizer(s) are constructed *after* calling this function,
-            then it is safe to omit this parameter. These optimizers will see the correct
-            model parameters.
-        replace_convs: replace strided :class:`torch.nn.Conv2d` modules with
+        model (Module): the model to modify in-place
+        replace_convs (bool, optional): replace strided :class:`torch.nn.Conv2d` modules with
             :class:`BlurConv2d` modules
-        replace_maxpools: replace eligible :class:`torch.nn.MaxPool2d` modules
+        replace_maxpools (bool, optional): replace eligible :class:`torch.nn.MaxPool2d` modules
             with :class:`BlurMaxPool2d` modules.
-        blur_first: for ``replace_convs``, blur input before the associated
+        blur_first (bool, optional): for ``replace_convs``, blur input before the associated
             convolution. When set to ``False``, the convolution is applied with
             a stride of 1 before the blurring, resulting in significant
             overhead (though more closely matching
             `the paper <http://proceedings.mlr.press/v97/zhang19a.html>`_).
             See :class:`~composer.algorithms.blurpool.BlurConv2d` for further discussion.
```

Comment:
Ah, I think these aren't linking since it needs the full path i.e. :class:`~composer.algorithms.blurpool.blurpool_layers.BlurConv2d`. @hanlint, is there a shortened form for this as well?

Reply:
yeah :class:`.BlurConv2d` should work as well, since that name is globally unique.